etc/httpd/global.conf

```diff
@@ -242,6 +242,9 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?download/?$ /$1firefox/new/ [L,R=301]
 # bug 957283
 RewriteRule ^/(pt-BR)/firefox/speed(/?)$ /b/$1/firefox/speed$2 [PT]
 
+# bug 1146982
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/send-to-device-post(/?)$ /b/$1firefox/send-to-device-post$2 [PT]
```

Comment:
that's more of a nit, not a blocker. It looks like the macro should always include the slash in the target, so it's a non-issue.